config/crd/bases/nifi.konpyutaika.com_nifinodegroupautoscalers.yaml

```diff
@@ -3,7 +3,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.8.0
+    controller-gen.kubebuilder.io/version: v0.9.0
```

Comment:
Did you intend for changes to this file to be included in this PR, or is this a result of an errant `make`? I've done the latter several times.

Reply:
Yes, this is an oversight following #120 . kubebuilder was upgraded in operator-sdk v1.22.1, so it should be included. Thank you for double checking! 👍

Reply:
I did not made the change, the change was done automatically by something when I did `make generate manifests`.